designer/server/src/main/scala/pl/touk/nussknacker/ui/process/deployment/DeploymentService.scala

```diff
@@ -128,8 +119,7 @@ class DeploymentService(
 
   protected def validateBeforeDeploy(
       processDetails: ScenarioWithDetailsEntity[CanonicalProcess],
-      deployedScenarioData: DeployedScenarioData,
```

Comment:
It was a part of `ProcessingTypeDeployedScenariosProvider` which is not needed now so it was removed.